openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KurentoHandler.java

```diff
@@ -337,7 +337,10 @@ JSONArray getTurnServers(final boolean test) {
 					mac.init(new SecretKeySpec(turnSecret.getBytes(), HMAC_SHA1_ALGORITHM));
 					StringBuilder user = new StringBuilder()
 							.append((test ? 60 : turnTtl * 60) + System.currentTimeMillis() / 1000L);
-					if (!Strings.isEmpty(turnUser)) {
+					final String uid = c.getUid();
```

Comment:
sorry for double-review wasn't concentrated enough :(((

later on you are calling this method with `c == null`
I would change this line to be 
`final String uid = c == null ? null : c.getUid();`